code/modules/modular_computers/hardware/job_disk.dm

```diff
@@ -68,6 +68,9 @@
 	if(disk_flags & DISK_STATUS)
 		progs_to_store += new /datum/computer_file/program/status(src)
 
+	if(disk_flags & DISK_SCI)
+		progs_to_store += new /datum/computer_file/program/science(src)
```

Comment:
God this is depressing

Reply:
constructive feedback where?

Reply:
I do not like disk_flags as a concept, it's this weird perversion of programs and it bugs me